setup-client.yml

```diff
@@ -101,6 +98,11 @@
       path: /usr/share/khotkeys/spectacle.khotkeys
       state: absent
 
+  - name: Menueeintrag fuer ADFC-Datenbank erstellen
+    template:
```

Comment:
Im Moment nutzt du die Template Funktion nicht. Man könnte auch copy nutzen https://docs.ansible.com/ansible/latest/modules/copy_module.html
dann müsste die Datei in das Verzeichnis files, statt templates

Reply:
Was genau meinst du damit? Der Task ist doch genau so aufgebaut, wie z.B. unter "- name: Erstelle /etc/netplan/02-adfc.yaml". Diese Datei wird auch aus dem Template-Ordner geholt.

Reply:
Der Unterschied ist, dasss bei netplan Variablen genutzt werden, da steht z.B.
```
 search: [{{ adfc_dns_search }}]
```

Und das wird ersetzt duch den Inhalt der Variable adfc_dns_search

Reply:
OK verstanden. Ist umgesetzt.